Mathlib/MeasureTheory/Function/L1Space/Integrable.lean

```diff
@@ -64,9 +64,11 @@ theorem Integrable.aestronglyMeasurable {f : α → ε} (hf : Integrable f μ) :
     AEStronglyMeasurable f μ :=
   hf.1
 
+open TopologicalSpace
```

Comment:
Can you just open this globally at the top of the file? (You also open it below, and I think it's good style to only open namespaces at the top of a file/section)